### ISSUE TITLE
Fails while compiling

### ISSUE THREAD

**Issue by 097115** [Titor8115/mandown#10]
macOS 10.11:

```
gcc -c -g -O3 -fPIC -Wall -Wsign-compare -Iparser -Iblender -Iinclude -I/usr/include/libxml2   -c -o src/mandown.o src/mandown.c
gcc -c -g -O3 -fPIC -Wall -Wsign-compare -Iparser -Iblender -Iinclude -I/usr/include/libxml2   -c -o src/view.o src/view.c
src/view.c:104:19: error: use of undeclared identifier
      'A_ITALIC'
        attr[1] = A_ITALIC;
                  ^
1 error generated.
make: *** [src/view.o] Error 1
```

**Comment by Titor8115** [comment]
Interesting, would you mind sharing your Ncurses version? I read somewhere that Homebrew has a new version of Ncursesw that links to the older Ncurses. I’ll do some search as well.

**Comment by 097115** [comment]
> Interesting, would you mind sharing your Ncurses version?

Was 6.1, upgraded with Homebrew to 6.2 (via downloading [this file](https://ftp.gnu.org/gnu/ncurses/ncurses-6.2.tar.gz)), same error :)

**Comment by Titor8115** [comment]
> > Interesting, would you mind sharing your Ncurses version?
> 
> Was 6.1, upgraded with Homebrew to 6.2 (via downloading [this file](https://ftp.gnu.org/gnu/ncurses/ncurses-6.2.tar.gz)), same error :)

**Comment by Titor8115** [comment]
- [ ]

**Comment by Titor8115** [comment]
> > Interesting, would you mind sharing your Ncurses version?
> 
> Was 6.1, upgraded with Homebrew to 6.2 (via downloading [this file](https://ftp.gnu.org/gnu/ncurses/ncurses-6.2.tar.gz)), same error :)

That is strange... I did find A_ITALIC macro in the tar file you uploaded. Would you mind try one more thing by changing A_ITALIC to A_NORMAL and see if it would compile.

**Comment by 097115** [comment]
Yeah, it compiles[1] but doesn't display italics as a consequence :)

-- 
[1]: I also had to change Makefile:30 from this:
    LDFLAGS=-g -O3 -Wl,--copy-dt-needed-entries -lncursesw -lxml2 -Wall -Werror
to this:
    LDFLAGS=-g -O3 -Wl,-lncurses -lxml2 -Wall -Werror

**Comment by Titor8115** [comment]
I’ll see what can be done.

The Makefile is under makeover. I didn’t expect the sudden increase in community and system at the beginning.

**Comment by 097115** [comment]
No problem :)

**Comment by lucamot** [comment]
On macOS, as a temporary fix, you could replace `A_ITALIC` with `NCURSES_BITS(1U,23)`, which is the original definition in ncurses.h. Also, remove `--copy-dt-needed-entries` from LDFLAGS as already stated above.

**Comment by 097115** [comment]
> you could replace A_ITALIC with NCURSES_BITS(1U,23)

@lucamot, the resulting file, compiled with these changes still doesn't display italics for me, though.

**Comment by Titor8115** [comment]
@097115 I uploaded a new Makefile and view.h to fix the compile issue and A_ITALIC issue. It just hit my mind that the terminal emulator might be the reason ITALIC isn't rendered. What emulator are you using?

**Comment by 097115** [comment]
> It just hit my mind that the terminal emulator might be the reason ITALIC isn't rendered

No :). I've tried iTerm and Alacritty, they both support italics.

(On this nw Makefile: it compiles now but italics indeed aren't rendered.)

**Comment by Titor8115** [comment]
Alright, I'll test it out if it is an issue across Mac OS. If that's case, I'll try to implement ANSI for the italics.
